app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,66 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[destroy]
+
+  # GET /recipe_foods/new
+  def new
+    @recipe_food = RecipeFood.new
+    @recipe_food.recipe_id = params[:recipe_id]
+    @foods = Food.all
+  end
+
+  # POST /recipe_foods or /recipe_foods.json
+  def create
+    @recipe_food = RecipeFood.new
+    @recipe_food.recipe_id = params[:recipe_id]
+    @recipe_food.food_id = params[:food_id]
+    @recipe_food.quantity = params[:quantity]
```

Comment:
- [ ] <b>OPTIONAL:</b> It is much better to use the secured way of passing params, something like this:
```rb
def create
    @recipe_food = RecipeFood.new(recipe_food_params)
   #.......
end

private

def recipe_food_params
    params.require(:recipe_food).permit(:recipe_id, : food_id, : quantity)
end